bad.php

```diff
@@ -0,0 +1,4 @@
+<?php
+function bad_braces() {
```

Comment:
Function name "bad_braces" is prefixed with a package name but does not begin with a capital letter